frontend/apps/design-system-storybook/package.json

```diff
@@ -31,22 +50,15 @@
     "@storybook/test-runner": "^0.19.1",
     "axe-playwright": "^2.0.3",
     "concurrently": "^9.1.0",
+    "eslint": "^9.16.0",
+    "globals": "^15.13.0",
     "http-server": "^14.1.1",
+    "prettier": "3.3.3",
```

Comment:
intentional to pin to 3.3.3?

Reply:
This seems to be the [recommendation from prettier](https://prettier.io/docs/en/install.html):

> Install an exact version of Prettier locally in your project. This makes sure that everyone in the project gets the exact same version of Prettier. Even a patch release of Prettier can result in slightly different formatting, so you wouldn’t want different team members using different versions and formatting each other’s changes back and forth.